javalib/src/main/scala/java/util/AbstractCollection.scala

```diff
@@ -0,0 +1,89 @@
+package java.util
+
+import scala.annotation.tailrec
+import scala.collection.immutable.{ List => SList }
```

Comment:
`{List => SList}` without spaces inside the `{}` (exceptional to the use of `{}` in imports)